src/ripple/app/ledger/LedgerTiming.h

```diff
@@ -52,67 +56,74 @@ std::uint32_t roundCloseTime (
 // they should not be changed arbitrarily.
 
 // The percentage threshold above which we can declare consensus.
-int const minimumConsensusPercentage = 80;
+auto constexpr minimumConsensusPercentage = 80;
 
+using namespace std::chrono_literals;
 // All possible close time resolutions. Values should not be duplicated.
-int const ledgerPossibleTimeResolutions[] = { 10, 20, 30, 60, 90, 120 };
+std::chrono::seconds constexpr ledgerPossibleTimeResolutions[] =
+    { 10s, 20s, 30s, 60s, 90s, 120s };
 
+#ifndef _MSC_VER
 // Initial resolution of ledger close time.
-int const ledgerDefaultTimeResolution = ledgerPossibleTimeResolutions[2];
+auto constexpr ledgerDefaultTimeResolution = ledgerPossibleTimeResolutions[2];
+#else
+// HH Remove this workaround of a VS bug when possible
+auto constexpr ledgerDefaultTimeResolution = 30s;
```

Comment:
I'll try it.  I actually had that in, and then took it back out because this is just a workaround, and I wasn't sure VS would swallow it.  I'll give it a go.